nailgun/entities.py

```diff
@@ -4966,6 +4969,28 @@ def __init__(self, server_config=None, **kwargs):
         super(PuppetModule, self).__init__(server_config, **kwargs)
 
 
+class Policies(Entity, EntityReadMixin,):
```

Comment:
Mind changing the class to `CompliancePolicies`? There could be many types of policies with more plugins.